pkg/tuple/hashing.go

```diff
@@ -42,7 +43,8 @@ func CanonicalBytes(rel Relationship) ([]byte, error) {
 
 	if rel.OptionalExpiration != nil {
 		sb.WriteString(" with $expiration:")
-		sb.WriteString(rel.OptionalExpiration.Format(expirationFormat))
+		truncated := rel.OptionalExpiration.UTC().Truncate(time.Second)
```

Comment:
why truncating to second resolution?

Reply:
CRDB doesn't store the nanos and it was breaking tests, so I truncated it for integrity purposes